### PR TITLE
feat: avatar and badge

### PR DESCRIPTION
**Pull request by yashpatel1797**
Added avatar and badge components in this commit. Utility class and landing page are pushed in the development branch.
[live link](https://thirsty-engelbart-dd2270.netlify.app/docs.html).

### CODE REVIEW

**Comment by parthpandyappp** [comment]
The Avatar component is as expected, but there's just a little suggestion that the heading `Avatar` should actually be a heading i.e. it should look a little bigger than other headings on the same page. Also, have some space between two different types of Avatar, I mean you should maintain a line-height or give some margin.

**Comment by yashpatel1797** [comment]
> The Avatar component is as expected, but there's just a little suggestion that the heading `Avatar` should actually be a heading i.e. it should look a little bigger than other headings on the same page. Also, have some space between two different types of Avatar, I mean you should maintain a line-height or give some margin.

Thanks for reviewing my code. 
1. I will change the heading after completing all the other components.
2. I have given 'gap' i will increase the value of gap.

File: components/Avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="author" content="Yash Patel" />
```
Comment (bharati-21) [inline components/Avatar/avatar.html:7]: Can add a description meta tag as well.

File: components/Avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    <title>Avatar</title>
+  </head>
+  <body>
+    <!-- avatar square -->
+    <h1>Avatar</h1>
+    <h1>Square avatar</h1>
+    <div class="avatar">
+      <img
+        src="https://picsum.photos/200"
+        alt="avatar"
```
Comment (bharati-21) [inline components/Avatar/avatar.html:19]: I suggest you could add different alt attributes for all images. E.g. Avatar extra small, Avatar small, Avatar large, etc.

File: components/Avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        class="avatar__round-border avatar-image-2xl"
+      />
+    </div>
+    <!-- avatar round -->
+    <h1>Round avatar</h1>
+    <div class="avatar">
+      <img
+        src="https://picsum.photos/200"
+        alt="avatar"
+        class="avatar__circle avatar-image-sm"
```
Comment (bharati-21) [inline components/Avatar/avatar.html]: Great class naming!

File: components/Badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  align-items: flex-end;
+  gap: 1rem;
+  justify-content: center;
+}
+.badge > div {
+  position: relative;
+}
+.num {
+  position: absolute;
+  border: var(--color-white) solid 2px;
```
Comment (bharati-21) [inline components/Badge/badge.css:13]: You could also just add `--white` for the variable name.

File: components/Badge/badge.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  display: inline-block;
+  bottom: 1.1rem;
+  padding: 0.6rem;
+  left: 4.6rem;
+}
+.avatar-online {
+  background-color: var(--color-primary-dark);
+}
+.avatar-away {
+  background-color: var(--color-yellow);
```
Comment (bharati-21) [inline components/Badge/badge.css:24]: A similar suggestion. `--yellow` is enough I guess.

**Review by bharati-21 — APPROVED** [review]
Great your class and variable naming. Neat, clean, and easy to read code.

Looks good to me!

File: components/Avatar/avatar.html (hunk 1 above)
Comment (yashpatel1797) [reply components/Avatar/avatar.html:7]: sure will do change in next commit.

File: components/Avatar/avatar.html (hunk 2 above)
Comment (yashpatel1797) [reply components/Avatar/avatar.html:19]: will do this also.

File: components/Avatar/avatar.html (hunk 3 above)
Comment (yashpatel1797) [reply components/Avatar/avatar.html]: Thanks

File: components/Badge/badge.css (hunk 4 above)
Comment (yashpatel1797) [reply components/Badge/badge.css:13]: initially, my plan was to have normal text white colour and background with different shade's of white but not added that so --color-bg-white something like this.

**Comment by yashpatel1797** [comment]
> The Avatar component is as expected, but there's just a little suggestion that the heading `Avatar` should actually be a heading i.e. it should look a little bigger than other headings on the same page. Also, have some space between two different types of Avatar, I mean you should maintain a line-height or give some margin.

Thanks for the review will increase the gap and heading font size.

**Review by vpress000 — APPROVED** [review]
Used gradient colors which entices to be good. You seemed good at choosing gradient colors.

**Review by parthpandyappp — COMMENTED** [review]
approved

**Pull request merged**